packages/itwin/tree-widget/src/components/trees/models-tree/ModelsTreeComponent.tsx

```diff
@@ -185,7 +185,7 @@ function View2DButton(props: ModelsTreeHeaderButtonProps) {
 
   useEffect(() => {
     setIs2dToggleActive(areAllModelsVisible(models2d, props.viewport));
-    return props.viewport.onViewedModelsChanged.addListener(() => setIs2dToggleActive(areAllModelsVisible(models2d, props.viewport)));
+    return props.viewport.onViewedModelsChanged.addListener(/* istanbul ignore next */() => setIs2dToggleActive(areAllModelsVisible(models2d, props.viewport)));
```

Comment:
why are we ignoring coverage here?

Reply:
Haven't found a way to test if setIs2dToggleActive was called.

Reply:
I believe this should be tested using react-testing-library with a test sounding like "updates button state when plan projection models display state changes":
1. initial state: all plan projection models are displayed
2. render button, confirm the button is active
3. change the models' display state, trigger the event
4. confirm button's state changed

Not saying that has to be done in this PR, but just don't think we should ignore coverage of this line.

Reply:
Added tests, which cover mentioned function.